### ISSUE TITLE
Filtering with multiple conditions ('and' and 'or') is not fully compatible in the GUI

### ISSUE THREAD

**Issue by dnk8n** [OneGraph/graphiql-explorer#73]
I noticed in raising a question on Stackoverflow, see https://stackoverflow.com/questions/69269367/hasura-how-to-filter-by-user-specified-ranges-where-data-contains-ranges/69271092#69271092 that the explorer is not fully functional in that it cannot produce the correct syntax for the filter I needed.

**Comment by sgrove** [comment]
I suspect this is because we haven't implemented support for lists yet - #32 

It's definitely doable, it's mainly an issue of figuring out the visual representation in such a small space.

**Comment by dnk8n** [comment]
Yeah, that looks like the cause. So this is a duplicate of #32 and could probably be closed and instead added as a comment, potentially.
